warehouse/oidc/services.py

```diff
@@ -287,11 +305,30 @@ def find_publisher(
             publisher = find_publisher_by_issuer(
                 self.db, self.issuer_url, signed_claims, pending=pending
             )
+
+            jwt_token_identifier: str | None = signed_claims.get("jti", None)
```

Comment:
Nitpick: explicit `None` default not needed:

```suggestion
            jwt_token_identifier: str | None = signed_claims.get("jti")
```